### ISSUE TITLE
Log at audit log when the receiver of a share deletes it

### ISSUE THREAD

**Issue by tflores** [nextcloud/server#18866]
**Is your feature request related to a problem? Please describe.**
Yes, because it would be important to log at audit log (app admin_audit) when the user who receives a share deletes it by himself.

**Describe the solution you'd like**
Including the following line in the function ```sharingHooks()``` in the file ```/nextcloud/apps/admin_audit/lib/AppInfo/Application.php```, seems to solve the problem:

```php
Util::connectHook(Share::class, 'post_unshareFromSelf', $shareActions, 'unshare');
```
**Describe alternatives you've considered**
No idea of alternatives.

**Additional context**
I've already tested this solution, but in an older version I had already installed and the results were ok.

**Comment by kesselb** [comment]
Thanks 👍

Mind to create a pull request with your changes?

**Comment by tflores** [comment]
@kesselb 
I'll read how to create a pull request and do it.
Thanks!

**Comment by szaimen** [comment]
Seems like this was implemented with #19236 -> closing.
